test/test_device_radix_sort.cu

```diff
@@ -1220,14 +1220,14 @@ int main(int argc, char** argv)
     int ptx_version = 0;
     CubDebugExit(PtxVersion(ptx_version));
 
-#ifdef QUICKER_TEST
+#ifdef CUB_TEST_MINIMAL
 
     enum {
         IS_DESCENDING   = false
     };
 
     // Compile/run basic CUB test
-    if (num_items < 0)      num_items       = 48000000;
+    if (num_items < 0)      num_items       = 24000000;
```

Comment:
This change looks unrelated. Was it intentional?

Reply:
This is intentional; that test just takes too long right now.